webapp/app/[locale]/stake/_components/stakeToast.tsx

```diff
@@ -0,0 +1,75 @@
+import { ExternalLink } from 'components/externalLink'
+import { CheckMark } from 'components/icons/checkMark'
+import { Link } from 'components/link'
+import { useChain } from 'hooks/useChain'
+import { useTranslations } from 'next-intl'
+import { useEffect, useState } from 'react'
+import { RemoteChain } from 'types/chain'
+import { CloseIcon } from 'ui-common/components/closeIcon'
+import { formatTxHash } from 'utils/format'
+import { type Hash } from 'viem'
+
+type Props = {
+  autoCloseMs?: number
+  chainId: RemoteChain['id']
+  txHash: Hash
+}
+
+export const StakeToast = function ({
+  autoCloseMs = 5000,
+  chainId,
+  txHash,
+}: Props) {
+  const [closedToast, setClosedToast] = useState(false)
+  const blockExplorer = useChain(chainId).blockExplorers.default
+
+  const t = useTranslations('stake-page')
+
+  useEffect(
+    function () {
+      if (autoCloseMs) {
+        const timer = setTimeout(function () {
+          setClosedToast(true)
+        }, autoCloseMs)
+
+        return () => clearTimeout(timer)
+      }
+      return undefined
+    },
+    [autoCloseMs],
+  )
+
+  if (closedToast) {
+    return null
+  }
+
+  return (
+    <div
+      className="shadow-soft fixed bottom-20 left-4 right-4 z-40 flex justify-between
+      gap-x-3 rounded-xl border border-solid border-black/85 bg-neutral-800 p-3.5
+    text-sm font-medium text-white md:bottom-auto md:left-auto md:right-8 md:top-20"
+    >
+      <div className="mt-[5px]">
```

Comment:
Let's prefer default tailwind classes instead of creating these custom ones, whenever possible

```suggestion
      <div className="mt-1">
```

or


```suggestion
      <div className="mt-1.5">
```

Reply:
Done!